<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageInterface</class>
 <widget class="QDialog" name="ImageInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Settings</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="4">
      <widget class="QSpinBox" name="imgHeight">
       <property name="toolTip">
        <string>Height of the image (in pixels)</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="imgWidth">
       <property name="toolTip">
        <string>Width of the image (in pixels)</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="ratioCheckBox">
       <property name="text">
        <string>Keep Ratio</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Oversampling</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="oversampling">
       <property name="toolTip">
        <string>Antialiases image (when larger then 1.0)</string>
       </property>
       <property name="prefix">
        <string>x </string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>64.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="color_comboBox">
     <item>
      <property name="text">
       <string>Use current background color</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use transparent background</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Choose background color</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="expandFrustum">
     <property name="toolTip">
      <string>When image aspect ratio differs from viewer's one, expand frustum as needed. Fits inside current frustum otherwise.</string>
     </property>
     <property name="text">
      <string>Expand Frustum if Needed</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ImageInterface</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ImageInterface</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
